### DISCUSSION TITLE
分享网址时如何自定义文字

### DISCUSSION THREAD

**Discussion by shuncon** [CaiJimmy/hugo-theme-stack#762] [Q&A] [answered]
### What happened?

分享网址时如何自定义文字

![image](https://user-images.githubusercontent.com/25509201/211997848-916dcc13-490c-4366-b779-1d536b44cf20.png)

如图所示，我想改为中文 需要修改哪里呢

### Hugo version

0.88

### Theme version

3.12.0

### What browsers are you seeing the problem on?

Chrome

### More information about the browser

win10,浏览器最新版

### Relevant log output

```shell
无日志无日志无日志无日志无日志无日志无日志
```

### Link to the page with bug

https://demo.stack.jimmycai.com

### Link to the source repository

https://github.com/CaiJimmy/hugo-theme-stack

**Comment by CaiJimmy** [comment, marked as answer]
模板逻辑在这里：https://github.com/CaiJimmy/hugo-theme-stack/blob/master/layouts/partials/data/description.html

默认使用页面的 `descriptions` 字段。

**Reply by shuncon** [reply]
我是要注释掉所有语句，还是如何呢？ 请见谅 我并不会编程

**Reply by shuncon** [reply]
而且应该支持自定义语句的吧，不应该只会显示hugo........等等

**Reply by shuncon** [reply]
已经解决，非常感谢
